src/datachain/cli/__init__.py

```diff
@@ -48,7 +48,6 @@ def main(argv: Optional[list[str]] = None) -> int:
     logger.setLevel(logging_level)
 
     client_config = {
-        "aws_endpoint_url": args.aws_endpoint_url,
```

Comment:
can we remove `aws_endpoint_url` now completely (from the client)? (can be a separate small PR)

Reply:
Why would we want that? client config is essentially how we pass the credentials config to datachain client for all other remotes as well from studio.